app/views/thesis/process_theses.html.erb

```diff
@@ -45,8 +45,18 @@
           --
           Status: <span class="thesis-status text-<%= thesis.css_alert_type %>">
             <%= thesis.status %></span>
-          <%# Other stuff should go here when e.g. people can mark as done or
-              a download link exists. %>
+
+          --
+
+          <br />
+          Files:  <% if thesis.files.attached? %>
+                    <% thesis.files.each do |file| %>
+                      <br />
+                       <%= link_to(file.filename, file.service_url) %>
```

Comment:
Is `file.filename` human-readable?

Reply:
Yeah, it's whatever the actual file was named the user chose to upload. so like "super cool file name.png"

Reply:
👍  `thesis.rev2.final.may18.FINAL.docx`